.vib/multus-cni/goss/multus-cni.yaml

```diff
@@ -15,6 +15,10 @@ file:
     exists: true
     filetype: symlink
     linked-to: /bitnami/multus-cni/host
+  /bitnami/multus-cni/host:
+    exists: true
+    filetype: directory
+    mode: "0775"
```

Comment:
Please use the Goss' template `check-directories.yaml` instead of adding a custom test here.